setup.py

```diff
@@ -19,10 +16,6 @@
     author_email="de-platform@kpn.com",
     url="https://github.com/kpn/py-pkgsettings",
     install_requires=[],
-    extras_require={
```

Comment:
Doesn't this cause, technically, a breaking change? I agree it is not supposed to be here at all.

Reply:
Formally speaking, yes, it would break someone who specified `pkgsettings[elasticsearch]` or `pkgsettings[all]`. I thought of that, but decided to go with `OPT`.

I can make it `{"elasticsearch": [], "all": []}`, no big deal